airbyte_cdk/test/entrypoint_wrapper.py

```diff
@@ -68,6 +69,7 @@ def __init__(
         uncaught_exception: Optional[BaseException] = None,
         *,
         message_file: Path | None = None,
+        completed_process: Optional[subprocess.CompletedProcess[str]] = None,
```

Comment:
Remove this from the constructor. Instead add a @classmethod factory function "from_completed_process()".